src/components/Forms/Button/index.js

```diff
@@ -38,8 +38,14 @@ export function Button({ children, className, size, ...other }) {
 
 export function InlineButton({ children, className, ...other }) {
   return (
-    <button className={cN(s.inlineButton, className)} {...other}>
+    <span
+      tabindex="0"
+      role="button"
+      aria-pressed="false"
+      className={cN(s.InlineButton, className)}
```

Comment:
class name is capitalized and that's why the link style is missing, it's not underlined and the cursor: pointer is missing